libraries/_executor.rb

```diff
@@ -143,6 +143,20 @@ def groovy(script)
       file.close! if file
     end
 
+
+
+    #
+    # Same as {Executor#groovy!}, but with the script file as parameter.
+    #
+    # @see groovy!
+    #
+    def groovyFile!(script)
+      scriptFile = File.open(script)
+      execute!("groovy #{scriptFile.path}")
+    ensure
+      scriptFile.close if scriptFile
+    end
```

Comment:
In Ruby programs, snake_case is preferred to camelCase for methods and variables. These should be

* `groovy_file`
* `script_file`

Respectively.